lib/DTLZ/hpo.py

```diff
@@ -0,0 +1,50 @@
+import os
+from importlib import import_module
+
+import time
+import numpy as np
+from deephyper.problem import HpProblem
+from deephyper.evaluator import profile, RunningJob
+from . import model as dtlz
+
+# Read DTLZ problem name and acquire pointer
+dtlz_prob = os.environ.get("DEEPHYPER_BENCHMARK_DTLZ_PROB", 2)
+dtlz_prob_name = f"dtlz{dtlz_prob}"
+dtlz_class_ptr = getattr(dtlz, dtlz_prob_name)
+
+# Read problem dims and definition (or read from ENV)
+nb_dim = int(os.environ.get("DEEPHYPER_BENCHMARK_NDIMS", 5))
+nb_obj = int(os.environ.get("DEEPHYPER_BENCHMARK_NOBJS", 2))
```

Comment:
I am wondering how we could be consistent between what should be "Environment variables" (e.g., `DEEPHYPER_BENCHMARK_NDIMS`) and "parameters of the `run`-function (e.g., `sleep=False`). Maybe the `sleep=False, sleep_mean=..., sleep_noise=...` should be implemented through a "decorator so that any function can be extended and we code it only once.